src/lib/components/Swap/Input.tsx

```diff
@@ -39,6 +43,13 @@ export default function Input({ disabled }: InputProps) {
   // extract eagerly in case of reversal
   usePrefetchCurrencyColor(swapInputCurrency)
 
+  const isRouteLoading = useMemo(
```

Comment:
Should this be `isTradeLoading`? Routes aren't really relevant in this file.

Reply:
Yeah isTradeLoading seems better